modules.yaml

```diff
@@ -155,8 +155,8 @@ modules:
     output: docs/stripeapplepay
     readme: StripeApplePay/README.md
   size_report:
-    max_compressed_size: 500
-    max_uncompressed_size: 1200
+    max_compressed_size: 700
+    max_uncompressed_size: 1700
```

Comment:
Does this PR increase the `StripeApplePay` pod size by ~40%? 😮

Reply:
Yeah, see below comment, this is because of `libswift_Concurrency.dylib` back-deployment. :( Fortunately it should only increase the size for apps targeting older iOS versions, though I'm not sure of the exact versions.

Reply:
I also considered putting this somewhere else, but we need to use it in FinancialConnections, Payments, and PaymentSheet, so there isn't a better location for it. We're planning to add more async support in 2025, so I think it's okay to move to it now.